clients/src/main/resources/common/message/ConsumerGroupDescribeResponse.json

```diff
@@ -0,0 +1,98 @@
+          "about": "The top-level error message, or null if there was no error." },
+        { "name": "GroupId", "type": "string", "versions": "0+", "entityType": "groupId",
+          "about": "The group ID string." },
+        { "name": "GroupState", "type": "string", "versions": "0+",
+          "about": "The group state string, or the empty string." },
+        { "name": "GroupEpoch", "type": "int32", "versions": "0+",
+          "about": "The group epoch." },
+        { "name": "AssignmentEpoch", "type": "int32", "versions": "0+",
+          "about": "The assignment epoch." },
+        { "name": "AssignorName", "type": "string", "versions": "0+",
+          "about": "The selected assignor." },
+        { "name": "Members", "type": "[]Member", "versions": "0+",
+          "about": "The members.",
+          "fields": [
+            { "name": "MemberId", "type": "uuid", "versions": "0+",
+              "about": "The member ID." },
+            { "name": "InstanceId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "The member instance ID." },
+            { "name": "RackId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "The member rack ID." },
+            { "name": "MemberEpoch", "type": "int32", "versions": "0+",
+              "about": "The current member epoch." },
+            { "name": "ClientId", "type": "string", "versions": "0+",
+              "about": "The client ID." },
+            { "name": "ClientHost", "type": "string", "versions": "0+",
+              "about": "The client host." },
+            { "name": "SubscribedTopicNames", "type": "[]string", "versions": "0+",
+              "about": "The subscribed topic names." },
+            { "name": "SubscribedTopicRegex", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+              "about": "the subscribed topic regex otherwise or null of not provided." },
+            { "name": "Assignment", "type": "Assignment", "versions": "0+",
+              "about": "The current assignment." },
+            { "name": "TargetAssignment", "type": "Assignment", "versions": "0+",
+              "about": "The target assignment." }
+            ]},
+        { "name": "AuthorizedOperations", "type": "int32", "versions": "3+", "default": "-2147483648",
+          "about": "32-bit bitfield to represent authorized operations for this group." }
+        ]}
+  ],
+  "commonStructs": [
+    { "name": "TopicPartitions", "versions": "0+", "fields": [
+      { "name": "TopicId", "type": "uuid", "versions": "0+",
+        "about": "The topic ID." },
+      { "name": "TopicName", "type":  "string", "versions": "0+",
+        "about":  "The topic name."},
+      { "name": "Partitions", "type": "[]int32", "versions": "0+",
+        "about": "The partitions." }
+    ]},
```

Comment:
i noticed you extracted this out from the Assignment struct as opposed to keeping it inline as in the KIP. Was it to make it more readable?

Reply:
Somehow I cant compile it when I inline the struct definition like it is in the KIP.
I get "java.lang.RuntimeException: Unable to locate a specification for the structure TopicPartitions".
Couldnt make any sense of this so I separated it which, which worked. I'm very happy for input on this one

Reply:
It seems that it doesn't like nested definitions. Looks good to me